lib/decimal.ex

```diff
@@ -2,8 +2,8 @@ defmodule Decimal do
   @moduledoc """
   Decimal arithmetic on arbitrary precision floating-point numbers.
 
-  A number is represented by a signed coefficient and exponent such that: `sign
-  * coefficient * 10^exponent`. All numbers are represented and calculated
+  A number is represented by a signed coefficient and exponent such that: ```sign
+  * coefficient * 10^exponent```. All numbers are represented and calculated
```

Comment:
Looks like a bug in the markdown parser. Please confirm if you can reproduce it in master of https://github.com/pragdave/earmark/, if so please report the bug.